src/login.ts

```diff
@@ -726,6 +726,9 @@ export const login = {
 
       const pages = await browser.pages();
       const page = pages[0];
+      await page.setExtraHTTPHeaders({
+        "Accept-Language": "en",
```

Comment:
Remove this comma
```suggestion
        "Accept-Language": "en"
```

Reply:
This comma was added by prettier as configured, you sure?

Reply:
Alright 👍